crates/bevy_input/src/system.rs

```diff
@@ -5,7 +5,12 @@ use crate::{
 use bevy_app::AppExit;
 use bevy_ecs::prelude::{EventReader, EventWriter};
 
-/// Sends the `AppExit` event whenever the "esc" key is pressed.
+/// Sends the [`AppExit`] event whenever the `ESC` key is pressed.
+///
+/// ## Note
+///
+/// This event is not used inside of `Bevy` by default. You can add the `exit_on_esc_system`
```

Comment:
```suggestion
/// This system is not added as part of `DefaultPlugins`; add it yourself if it is desired.

Reply:
I updated the description, but I'm kind of thinking about removing the whole system. It doesn't really make sense to me to have something like this embedded into the bevy source code.

The system is hard coded to quit on `ESC` and can't be easily changed by users. So if someone wanted to change the system to use `Space` to quit you would have to copy paste the whole system into your project anyway. So I would suggest we move it to an example or a doc comment somewhere.

If we don't delete it, it would make more sense to use the `Input<KeyCode>` resource instead of the `KeyboardInput` events to make the system simpler and faster.